llama-index-integrations/llms/llama-index-llms-huggingface-api/llama_index/llms/huggingface_api/base.py

```diff
@@ -34,7 +34,7 @@ def chat_messages_to_conversational_kwargs(
         raise NotImplementedError("Messages passed in must be of odd length.")
     last_message = messages[-1]
     kwargs: Dict[str, Any] = {
-        "text": last_message.content,
+        "text": last_message,
         **last_message.additional_kwargs,
```

Comment:
If we follow the types, this chat messages always have a .content 

If this is causing an issue that means the caller isn't passing in the proper types